app/views/dashboards/_modal.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="modal fade new-call" tabindex="-1" role="dialog">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-body">
+        <div class="calls-layout">
+          <button type="button" class="close close-terms" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+          <h4 class="modal-title calls-request">
```

Comment:
As noted above, would love to move this h4 into the other partial. Can replace it with a `div` so the javascript still works, I think?